webpack.config.js

```diff
@@ -99,6 +99,7 @@ const common = {
         },
         modulesDirectories: ['node_modules', 'vendor_modules'],
         root: [p.kendo],
+        extensions: ['', '.js', '.jsx'], // allows importing these file types without extensions
```

Comment:
i wonder if it's better to not have extensions.  it's wierd that we don't add extensions for JS, but we do add them for other file types....  but for backwards compat i guess we should just leave .js alone